src/neuroconv/tools/nwb_helpers/_metadata_and_file_helpers.py

```diff
@@ -370,3 +375,46 @@ def configure_and_write_nwbfile(
 
     with IO(output_filepath, mode="w") as io:
         io.write(nwbfile)
+
+
+def configure_and_export_nwbfile(
+    nwbfile: NWBFile,
+    export_nwbfile_path: Path,
+    backend_configuration: BackendConfiguration,
+) -> None:
+    configure_backend(nwbfile=nwbfile, backend_configuration=backend_configuration)
+
+    IO = BACKEND_NWB_IO[backend_configuration.backend]
+    nwbfile.set_modified()
+    with IO(export_nwbfile_path, mode="w") as io:
+        io.export(nwbfile=nwbfile, src_io=nwbfile.read_io, write_args=dict(link_data=False))
+
+
+def repack_nwbfile(
+    *,
+    nwbfile_path: Path,
+    export_nwbfile_path: Path,
+    backend: Literal["hdf5", "zarr"] = "hdf5",
+    export_backend: Literal["hdf5", "zarr", None] = None,
+    use_default_backend_configuration: bool = True,
+    backend_configuration_changes: dict = None,
+):
+    """Repack the NWBFile with the new backend configuration changes."""
+    IO = BACKEND_NWB_IO[backend]
+    with IO(nwbfile_path, mode="r") as io:
+        nwbfile = io.read()
+        if use_default_backend_configuration:
+            backend_configuration = get_default_backend_configuration(nwbfile=nwbfile, backend=backend)
+        else:
+            backend_configuration = get_existing_backend_configuration(nwbfile=nwbfile)
+        dataset_configurations = backend_configuration.dataset_configurations
+
+        backend_configuration_changes = backend_configuration_changes or dict()
```

Comment:
minor stylistic point: I would consider this line input parsing and would generally put it first in the function,- here, before the line `IO = BACKEND_NWB_IO[backend]`. Another reason is this line does not need to be in the context it is in, so it should be moved outside. If you can move a line of code outside of an indent, you usually should.